css-fonts-and-colors/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Montserrat, sans-serif;
+  background-color: aliceblue;
+}
```

Comment:
We only want the `section` here to be of a light blue color. If we apply to the `body` the whole page will be the light blue color but what if that's not the default color we want on the page and just want the section to stick out?